packages/table/test/columnTests.tsx

```diff
@@ -84,4 +84,15 @@ describe("Column", () => {
         const col2cells = table.element.queryAll(`.${Classes.columnCellIndexClass(2)}`);
         col2cells.forEach((cell) => expectCellLoading(cell, CellType.BODY_CELL, false));
     });
+
+    it("passes custom class name to renderer", () => {
+        const CLASS_NAME = "my-custom-class-name";
+        const table = harness.mount(
+            <Table numRows={5}>
+                <Column className={CLASS_NAME} />
+            </Table>,
+        );
+        const columnNode = table.find(`.${Classes.TABLE_HEADER}`, 0).element;
+        expect(columnNode.classList.contains(CLASS_NAME)).to.be.true;
```

Comment:
True, but that function has been wholly unreliable for me for some reason. I'll give it a try though.